java/vector/src/test/java/org/apache/arrow/vector/TestDictionaryVector.java

```diff
@@ -982,15 +981,15 @@ public void testDictionaryUIntOverflow() {
     }
   }
 
-  private int[] convertListToIntArray(JsonStringArrayList list) {
+  private int[] convertListToIntArray(List list) {
     int[] values = new int[list.size()];
     for (int i = 0; i < list.size(); i++) {
       values[i] = (int) list.get(i);
     }
     return values;
   }
 
-  private Object[] convertMapValuesToArray(JsonStringHashMap map) {
+  private Object[] convertMapValuesToArray(Map map) {
```

Comment:
should this be Map<?, ?>?

Reply:
Yes, I think so. A Map should be enough here?